Только в /root/dk_scripts/tmp/2: 6-Feb-2016115423LINK-W-Downfa33KTCS1-SW3-48_192.168.22.75.conf
Только в /root/dk_scripts/tmp/2: 8-Jan-2012183116LINK-W-Downfa34KTCS2-SW1-48_192.168.22.84.conf
diff -r /root/dk_scripts/tmp/2/LBR31-SW4-48_192.168.22.203.conf /root/dk_scripts/tmp/1/LBR31-SW4-48_192.168.22.203.conf
47a48,50
> interface fastethernet18
> negotiation 100h 100f 
> exit
diff -r /root/dk_scripts/tmp/2/sw-east-1-3_192.168.200.243.conf /root/dk_scripts/tmp/1/sw-east-1-3_192.168.200.243.conf
181d180
<  switchport access vlan 30 
Только в /root/dk_scripts/tmp/1: TCS1-SW3-48_192.168.22.75.conf
Только в /root/dk_scripts/tmp/1: TCS2-SW1-48_192.168.22.84.conf
Только в /root/dk_scripts/tmp/1: TCS4-SW1-48_192.168.22.71.conf
